patroni/postgresql.py

```diff
@@ -657,7 +662,10 @@ def rewind(self, r):
         dsn = 'user={user} host={host} port={port} dbname={database} sslmode=prefer sslcompression=1'.format(**r)
         logger.info('running pg_rewind from %s', dsn)
         try:
-            return subprocess.call(['pg_rewind', '-D', self._data_dir, '--source-server', dsn], env=env) == 0
+            return subprocess.call([self._pgcommand('pg_rewind'),
+                                    '-D', self._data_dir,
+                                    '--source-server', dsn,
+                                   ], env=env) == 0
```

Comment:
flake8 still complains:
patroni/postgresql.py:668:36: E124 closing bracket does not match visual indentation

You can check it locally by running `python setup.py flake8`